docs/prometheus.md

```diff
@@ -0,0 +1,21 @@
+## syzkaller metrics
```

Comment:
Since this is the article name, it would be reasonable to start with with single `#`.
Also everything in this repository is syzkaller-something. So prefixing every title with "syzkaller" is pointless. It's "Prometheus" what's non-obvious/non-default here, so I would call it:
```
# Prometheus metrics
```
or something like "Execution metrics" (but then the file need to be renamed as well).